### ISSUE TITLE
Review Scroll Viewport Features to Implement on Platform Website 

### ISSUE THREAD

**Issue by johnny-jesensky-adhoc** [department-of-veterans-affairs/va.gov-team#70450]
## Info
After the spike performed in Sprint 40, it was discovered that there is quite a lot of out-the-box functionality in ScrollViewport that we can leverage, and implement with minimal effort. This ticket is to outline some of the easy to implement features, that would help resolve some of the findability issues that were uncovered in the study over the summer.

## Tasks
- [x] Create list of features in Scroll Viewport, with rationale as to the benefit
- [x] Review features with Erika
- [ ] Make decision on which ones we'd want to implement.

## Acceptance Criteria

- [x] Feature list created.
- [x] Scroll Viewport features have been reviewed with Erika.

## Artifacts
[Sprint 41_ Scroll Viewport Functionality (#70450).pptx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN3dhQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--fb82d1c1ef31099e0b4e89012f56a72d004d604c/Sprint%2041_%20Scroll%20Viewport%20Functionality%20%28%2370450%29.pptx)

**Comment by EWashb** [comment]
These are all fantastic suggestions, @johnny-jesensky-adhoc. I appreciate you thoroughly examining the options to improve our website's user experience. 

Something to consider on the header and footer recommendations: we will likely need to sync up with the Design System team if the intent is to replicate what is currently surfaced on va.gov. Those components require alignment with the [VA.gov Design System](https://design.va.gov/components/). This doesn't mean we can't or shouldn't do this, but it expands our scope a bit. 

Another option would be to determine what a header and footer, as you recommended, may look like if they aren't 1:1 of the larger site. Especially considering our audience is different, we could do some additional discovery on what a successful version of a header and footer would be for our Platform website. 

For the other feature recommendations (1, 2, and 5), I don't see the harm in doing those now. We can follow up those enhancements with another round of research after they are launched. 

Similarly, I want to see how our site aligns with [VA Section 508 and accessibility standards.](https://digital.va.gov/web-governance/standards-and-policies/accessibility/) I wonder if there has ever been an audit of our site to see if we are compliant. An accessibility audit could be another good step in aligning our standards with those across OCTO. 

I am happy to discuss these more next week! Again, this is wonderful work.

**Comment by johnny-jesensky-adhoc** [comment]
Hi @EWashb, thank you for this feedback, (and apologize for the delay in response here)! In response to your comments above:

**- For Header & Footer Links:** I agree that some extra discovery might be beneficial, so we have some sound rationale and data to rely on before making any changes.  I'm curious to know what VFS Team members would find most valuable, as well as what the rest of the Platform Team would prefer regarding links they might want to highlight. The same is true for landing pages. For a next step on this, I can work to build out a proposed research plan, that could get us to at least a Version 1 of new navigation.

**_- For Features 1, 2, 5_** - I agree in that they are so trivial/low risk to implement. If you are ok with it, I can work to enable those next sprint (Sprint 42). 

_**- 508 and Accessibility Standards**_ - agree, and this is one of the epics I'd like to propose for next year. I learned that Scroll Viewport is putting a priority on making their sites more accessible, but there are some things directly within our control. We are able to adjust the look and feel of the site through some Custom CSS that we can inject, (another feature I discovered in my spike), as well as checking for alt-text, so think having a designated epic towards this would help us move the needle on this.

Happy to discuss all of this in more detail when we connect next over a call.
